helsingborg-stad/app-mitt-helsingborg
# Chore/cleanup-xcworkspace

## Explain the changes you’ve made

Cleaned up some of the ios project files.

## Explain why these changes are made

General cleanup. Duplicate xcworkspace caused some confusing as well.

## Explain your solution

[The run that made it](https://github.com/helsingborg-stad/app-mitt-helsingborg/actions/runs/3198195441/jobs/5222526925).

## How to test

1. Checkout this branch
2. Verify `yarn ios` and building work as expected.

## Tested environments

- [] Storybook on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.
